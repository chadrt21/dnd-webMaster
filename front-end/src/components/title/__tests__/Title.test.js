import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Title from '../index';

configure({ adapter: new Adapter() });

describe(
	'TitleComponent',
	() => {
		it('should render the component', () => {
			const component = shallow(
				<Title>Here is the title</Title>
			);
			expect(component).toMatchSnapshot();
		})
	}
);
