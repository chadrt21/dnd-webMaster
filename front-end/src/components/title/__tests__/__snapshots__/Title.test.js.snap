// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleComponent should render the component 1`] = `ShallowWrapper {}`;
